slides/index.html

```diff
@@ -20,11 +20,23 @@
 
 ## Hvem er vi?
 
-Studerer datateknikk på NTNU
+???
+
+- Først og fremst takk til fagkom for gratis muffins og kaffe
+- 4.-5. klasse data@NTNU
+- Blåtannfrelste personligheter
+- Laget et open source blåtannbibliotek for mbed i forbindelse med sommerjobb hos Nordic Semiconductor
+
+---
+class: center, middle
+
+## Hvem var Harald Blåtann?
```

Comment:
Maybe better with just "Harald Blåtann", and then a huge fullscreened background image of him looking all fierce-like.